geo/fuzz/fuzz_targets/simplify.rs

```diff
@@ -0,0 +1,11 @@
+#![no_main]
+
+use geo::algorithm::simplify::Simplify;
+use libfuzzer_sys::fuzz_target;
+
+fuzz_target!(|tuple: (geo_types::Polygon<f32>, f32)| {
+    if tuple.1 < 0. {
+        return;
+    }
+    tuple.0.simplify(&tuple.1);
```

Comment:
So basically, we're only testing that no `panic` occurs. Is that right? That seems helpful.

Do you think it's worth checking additional invariants?

I couldn't come up with much... maybe:
- number of coords in the output <= input's
- the euclidean length of the output's linestrings should be <= input's

I'd also be happy to leave it as is.

Reply:
https://github.com/georust/geo/pull/633/commits/dd4139d2d841dad4378725ae599c242332570d66